content/blog/2021/04/20-jenkins-contributor-awards.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Jenkins Contributor Awards - Nominations Open"
+tags:
+- awards
+- jenkins
+- cdcon
+author: cdfoundation
+description: "Nominate someone for the Jenkins Contributor Awards. Deadline is April 30."
+opengraph:
+  image: /images/post-images/2021/jenkins-awards-2021.png
```

Comment:
Image is missing

Reply:
Yes, I can't upload it because my fork branch isn't up to date and I tried to do it with Gitpod but it didn't work :D and I don't know how to with the command line. So I put the image in the pull request comment